frontend/magic_links/routes.py

```diff
@@ -120,6 +122,13 @@ def new():
     # Grabbing fund and round info from query params and validating
     fund_short_name = request.args.get("fund")
     round_short_name = request.args.get("round")
+    govuk_notify_reference = request.args.get("govuk-notify-reference", None)
```

Comment:
Unless we don't control the producer of this 🙇 `s/-/_/g`

Reply:
`s/-/_/g` ew